components/x-teaser/src/Title.jsx

```diff
@@ -16,8 +16,11 @@ export default ({ title, altTitle, headlineTesting, relativeUrl, url, indicators
 				{displayTitle}
 			</Link>
 			{indicators && indicators.accessLevel === 'premium' ? (
-				<span className={premiumClass} aria-label="Premium content">
-					Premium
+				<span>
```

Comment:
Is the outer span required?

Reply:
It seems so, throws an error without, which I think is to do with jsx not allowing adjsent elements.

Reply:
Ah, boo.

Reply:
you can in React with a `<Fragment>`, but that's not supported in stable Preact, so we can't use it in x-dash yet

Reply:
@i-like-robots happy to take a recommendation on a better way.

Reply:
It's fine the way it is, no need to delay!